docs/pages/server-access/getting-started.mdx

```diff
@@ -69,47 +70,72 @@ server.
    This helps to minimize the direct sharing of tokens even when they are dynamically generated.
 </Admonition>
 
-### Join your Node to the cluster
+### Join your SSH Service to the cluster
 
-On your Node, save `token.file` to an appropriate, secure, directory you have
+On your SSH Service machine, save `token.file` to an appropriate, secure, directory you have
 the rights and access to read.
 
 <ScopedBlock scope={["oss", "enterprise"]}>
 
-Start the Node. Change `tele.example.com` to the address of your Teleport Proxy
+Configure the SSH Service instance. Change `tele.example.com` to the address of your Teleport Proxy
 Service. Assign the `--token` flag to the path where you saved
 `token.file`.
 
 ```code
-# Join cluster
-$ sudo teleport start \
-   --roles=node \
+# Generate the Teleport configuration file, which is written to /etc/teleport.yaml
+$ sudo teleport node configure \
+   --labels=env=example \
    --token=/path/to/token.file \
-   --auth-server=tele.example.com:443
+   --proxy=tele.example.com:443 \
+   -o file
 ```
 
 </ScopedBlock>
 <ScopedBlock scope={["cloud"]}>
 
-Start the Node. Change `mytenant.teleport.sh` to your Teleport Cloud tenant
+Configure the SSH Service instance. Change `mytenant.teleport.sh` to your Teleport Cloud tenant
 address. Assign the `--token` flag to the path where you saved `token.file`.
 
 ```code
-# Join cluster
-$ sudo teleport start \
-   --roles=node \
+# Generate the Teleport configuration file, which is written to /etc/teleport.yaml
+$ sudo teleport node configure \
+   --labels=env=example \
    --token=/path/to/token.file \
-   --auth-server=mytenant.teleport.sh:443
+   --proxy=mytenant.teleport.sh:443 \
+   -o file
 ```
 
 </ScopedBlock>
 
+### Start the Teleport SSH Service
+
+On the host where you will run the Teleport SSH Service, execute the
+following command, depending on whether you installed Teleport using a package
+manager or via a TAR archive:
+
+<Tabs>
+<TabItem label="Package Manager">
+
+```code
+$ sudo systemctl start teleport
+```
+</TabItem>
+<TabItem label="TAR archive">
+
+```code
+$ sudo teleport install systemd --output=/etc/systemd/system/teleport.service;
+$ sudo systemctl enable teleport;
+$ sudo systemctl start teleport;
+```
+</TabItem>
+</Tabs>
+
 ### Access the Web UI
 
 Run the following command to create a user that can access the Teleport Web UI:
 
 ```code
-$ sudo tctl users add tele-admin --roles=editor,access --logins=root,ubuntu,ec2-user
+$ tctl users add tele-admin --roles=editor,access --logins=root,ubuntu,ec2-user
```

Comment:
Are we confident that sudo won't be required here?

Reply:
Yes, that's only required if you're connecting directly on the auth server.  The other db access guides like https://goteleport.com/docs/database-access/getting-started/ doesn't use `sudo`.